src/test/java/nl/beehivejava/scriptj/io/FileResourceLoaderTest.java

```diff
@@ -0,0 +1,14 @@
+package nl.beehivejava.scriptj.io;
+
+import nl.beehivejava.scriptj.util.builder.ObjectBuilders;
+
+/**
+ * @author Lesley
+ */
+public final class FileResourceLoaderTest extends ResourceLoaderTest {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2187)